mne/io/nirx/tests/test_nirx.py

```diff
@@ -78,6 +78,52 @@ def test_nirx():
     assert raw.annotations[1]['description'] == '2.0'
     assert raw.annotations[2]['description'] == '1.0'
 
+    # Test location of detectors
+    # The locations of detectors can be seen in the first
+    # figure on this page...
+    # https://github.com/mne-tools/mne-testing-data/pull/51
+    # And have been manually copied below
+    # These values were reported in mm, but according to this page...
+    # https://mne.tools/stable/auto_tutorials/intro/plot_40_sensor_locations.html
+    # 3d locations should be specified in meters, so that's what's tested below
```

Comment:
The other thing is that `ch['loc']` should be in head coordinates if possible for consistency. Head coords are defined by the Nasion/LPA/RPA in MNE.

fsaverage is in MNI coords and we have the `fsaverage-fiducials.fif` file in `mne/data/fsaverage`. And the head<->MRI trans in `fsaverage-trans.fif`. So the best thing to do here is probably to transform all of these locations so that they are in Neuromag head coords, and also append the fiducials in `info['dig']`. Then when you do `plot_alignment(raw.info, subject='fsaverage', dig=True, trans=fsaverage_trans)` everything will look correct.

Reply:
... if this does not make sense, I can try pushing some commits quickly to do this.

Reply:
Haha I am madly reading [this](https://mne.tools/stable/overview/implementation.html?highlight=neuromag#the-forward-solution) trying to make sense of all these different coordinate systems. If you can quickly push something easily then please go ahead. Otherwise, I will eventually get around to it.

Reply:
Will do